### ISSUE TITLE
Improve header layout on mobile

### ISSUE THREAD

**Issue by bkeepers** [probot/probot.github.io#63]
The header on the mobile layouts needs a little love. I'm open to ideas for how to improve it. 

If you're looking for recommendations, I would consider just hiding the "PROBOT" text, and having the nav links on the same line, maybe aligned to the right.

<img width="310" alt="featured_apps___probot" src="https://user-images.githubusercontent.com/173/30940601-38655f10-a3a7-11e7-9ebf-da691d0d8c1b.png">

## How

The website uses [primer](http://primercss.io), which is GitHub's 😻 CSS toolkit. It's amazing, but docs are currently being revamped, and thus not entirely up to date.

For hiding things, try adding the`d-sm-none` CSS class to the element, which will make elements disappear on mobile.

```html
<p class="d-sm-none">And for my next trick, I will make this paragraph disappear on your mobile device.</p>
```

Feel free to ask questions about primer here or on [Slack](https://probot-slackin.herokuapp.com/)

**Comment by Gutem** [comment]
I do it

**Comment by Pratham567** [comment]
I can assist you in this..

**Comment by bkeepers** [comment]
If you want to claim this, please just open a pull request for it.

**Comment by Pratham567** [comment]
OKk

**Comment by JasonEtco** [comment]
Closed by #69
